playwright-tests/tests/editor.spec.js

```diff
@@ -276,4 +276,84 @@ test.describe("?page=projects&tab=editor", () => {
       await expect(warningText).toBeVisible();
     });
   });
+
+  test.describe("Import project from NEAR Catalog", () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto(`/${ROOT_SRC}?page=projects&tab=catalogImport`);
+    });
+    test.use({
+      storageState: "playwright-tests/storage-states/wallet-connected.json",
+    });
+
+    // should have visible projects
+    test("should have visible projects", async ({ page }) => {
+      const projectHeading = page.getByRole("heading", {
+        name: "Race of Sloths",
+      });
+      await expect(projectHeading).toBeVisible({ timeout: 60000 });
+    });
+
+    test("Should be able to create a new project from NEAR Catalog", async ({
+      page,
+    }) => {
+      const projectHeading = page.getByRole("heading", {
+        name: "Race of Sloths",
+      });
+      await expect(projectHeading).toBeVisible({ timeout: 60000 });
+
+      const parentDiv = page
+        .locator("div.project-card")
+        .filter({ has: projectHeading });
+      await expect(parentDiv).toBeVisible({ timeout: 60000 });
+      await parentDiv.dispatchEvent("click");
+
+      const nextButton = page.getByRole("button", { name: "Next" });
+      await expect(nextButton).toBeVisible({ timeout: 60000 });
```

Comment:
All these 60 second time outs... remove all of them. If test is failing because of 60 second time out somewhere, it's a bad flaky test

Reply:
Haha, I only added these to make it work on my machine. Forgot to remove them.